packages/selinux-policy/object.cil

```diff
@@ -135,7 +140,7 @@
 
 ; Dynamic objects are files on temporary storage with special rules.
 (typeattribute dynamic_o)
-(typeattributeset dynamic_o (etc_t))
+(typeattributeset dynamic_o (etc_t binfmt_misc_fs_t))
```

Comment:
`dynamic_o` might be one of the worst names in the policy, but it has the intended set of "never allow" rules already.